dice-game-ui-angular/src/app/main/main.component.ts

```diff
@@ -3,13 +3,10 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-main',
   templateUrl: './main.component.html',
-  styleUrls: ['./main.component.scss']
+  styleUrls: ['./main.component.scss'],
 })
 export class MainComponent implements OnInit {
+  constructor() {}
```

Comment:
Both `constructor` and `ngOnInit` are not needed at all and can be removed,

Reply:
Removed